### ISSUE TITLE
LossRecord

### ISSUE THREAD

**Issue by LibertFan** [Wentong-DST/im2p#2]
Is the figure of loss really the training loss ? It seems too small.
Is your model loss calculated according to the paper ? Or you calculate it in word level instead of paragraph level ?

**Comment by Wentong-DST** [comment]
Hi LibertFan,
The loss is the training loss, and the validation loss is higher than it. So obviously there is an overfitting problem. I think the loss is calculated in word level.
The original repo author has ceased any support, so I would recommend you to move onto other image captioning models. 
Please check [Luo Ruotian's implementation of self-critical model](https://github.com/ruotianluo/self-critical.pytorch).
